package.json

```diff
@@ -11,7 +11,7 @@
     "type": "git",
     "url": "https://github.com/mohsenbostan/nestjs-grpc-exceptions.git"
   },
-  "version": "0.2.0",
+  "version": "0.2.1",
```

Comment:
please revert this one back to `0.2.0`. this will be updated automatically after release.

Reply:
Hi there @mohsenbostan

Thx for your review and pointing that out - I will revert the version :)